code/src/reducers/tasks.js

```diff
@@ -0,0 +1,61 @@
+// src/reducers/tasks.js
+import { createSlice } from "@reduxjs/toolkit";
+
+export const tasks = createSlice({
+  name: "tasks",
+  initialState: {
+    tasks: [
+      {
+        id: 1,
+        text: "Watch video on actions & reducers",
+        added: Date.now() - 604800000,
```

Comment:
Complicated! Haha.

If this was in a project for a company, I'd want to see a comment explaining what this is to save me doing the maths. Or, since you have moment, you could just use that to make it more readable:

```js
added: moment().subtract(4, 'days')
```